# SetReward: Updates the learner with a reward

# The state of the scene and command (un)validated
SceneState scene_state
UserCommand command
bool good
---
# No return parameter

